assets/app/scripts/services/data.js

```diff
@@ -708,7 +706,7 @@ angular.module('openshiftConsole')
 
   DataService.prototype._urlForType = function(type, id, context, isWebsocket, params) {
```

Comment:
yeah... on second though, we probably don't need caller... shouldn't be hard to track down a call to a data service method with a mixed case type if needed